packages/web-components/examples/codesandbox/components/modal/index.html

```diff
@@ -27,19 +27,19 @@ <h1>Hello World! 👋</h1>
     <div id="app">
```

Comment:
@asudoh I think the folder name should change too to be consistent.

Reply:
Great catch @jeffchew - Fixed.